README.md

```diff
@@ -1,24 +1,61 @@
-# README
+# Recipe-app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app allow you to make them public so anyone can access them.
 
```

Comment:
- [ ] **OPTIONAL**
It'd be great if you included a couple of images of your app in the README file for a nicer appearance. Also, a live demo link would be even greater 😉